package.json

```diff
@@ -46,6 +46,7 @@
     "lodash": "3.10.1",
     "react": "15.3.2",
     "react-native": "0.34.1",
+    "react-native-parallax-scroll-view": "^0.19.0",
```

Comment:
pleasantly simple, with no extra dependencies in the lockfile, worked out pretty well so far